src/views/Scan.vue

```diff
@@ -238,6 +262,25 @@
 			closeFilters() {
 				this.filtersVisible = false;
 			},
+			detectFullscreen: () => {
```

Comment:
Why not just use a one-liner like `return document.msFullscreenElement || document.webkitFullscreenElement || document.fullscreenElement`. We don't really have guidelines for this, but since this function returns a boolean, it should be called something like `fullscreenAvailable` and moved from `methods` to `computed`.

Reply:
Hi @t11r , thanks for your feedback. Unfortunately, I can’t use this as document.fullscreenElement and its variants don’t return a boolean; they either return null or the DOM element that is currently fullscreen so document.fullscreenElement would always be returned as none of the previous values would ever be true. There were functions in the Fullscreen API which detected whether a browser could support fullscreen (and these returned a boolean) but they’ve since been deprecated in favour of the ones I’ve used.

Reply:
I referred to the return value being cast to boolean, but I missed that this function is also used to determine if fullscreen is currently active. Maybe add a comment about the meaning of the different return values.